test/test_files/log_files/samus_returns/vanilla_game.rdvgame

```diff
@@ -1,622 +0,0 @@
-{
```

Comment:
unsure if out of scope, but this rdvgame should not be completely deleted, but instead be used as resolver verification.

Reply:
It's unused. If anyone has the need to use it, they are free to add it again.